Detectors/ITSMFT/ITS/simulation/src/Digitizer.cxx

```diff
@@ -89,7 +89,7 @@ DigitContainer& Digitizer::process(TClonesArray* points)
   // Convert points to digits
   const SegmentationPixel* seg = (SegmentationPixel*)mGeometry.getSegmentationById(0);
   for (TIter pointiter = TIter(points).Begin(); pointiter != TIter::End(); ++pointiter) {
-    Point* point = dynamic_cast<Point*>(*pointiter);
+    Hit* point = dynamic_cast<Hit*>(*pointiter);
```

Comment:
unrelated to this commit: consider using a static cast here for performance reasons.